app/AttributionGenerator.js

```diff
@@ -158,7 +158,9 @@ $.extend( AttributionGenerator.prototype, {
 		// Prefer attribution over author:
 		var $attribution = this._asset.getAttribution();
 		if( $attribution ) {
-			return format === 'html' ? $attribution : $attribution.text();
+			if( $attribution.length && $attribution.text().length ) {
```

Comment:
Using `if ( $jQueryCollection.length )` like it is a boolean is common practice. Not so much for the text. I would prefer an `!== ''` check there.